sdks/java/testing/load-tests/build.gradle

```diff
@@ -63,13 +63,17 @@ dependencies {
   compile library.java.kafka_clients
 
   compile project(path: ":sdks:java:core", configuration: "shadow")
-  compile project(path: ":runners:direct-java", configuration: "shadow")
   compile project(":sdks:java:io:synthetic")
   compile project(":sdks:java:testing:test-utils")
   compile project(":sdks:java:io:google-cloud-platform")
   compile project(":sdks:java:io:kafka")
   compile project(":sdks:java:io:kinesis")
-
```

Comment:
there should be an empty line between "compile" and "gradleRun" lines - like there was before